content/practices/research-workshop/index.md

```diff
@@ -0,0 +1,128 @@
+---
+title: "Research Workshop"
+linkTitle: "Research Workshop"
+description: "Level-set on user research best practices"
+tags: ["Discovery"]
+length: "45 mins - 1 hour"
+participants: "Core team"
+# custom "cover" image example: "boris/boris.png"
+image: "default-cover.png" 
+lastmod: "2021-05-14"
+why: 
+- Continuous research helps ensure that we’re building something of value at each step. Needs and problems live with people, so in order to create value we need a deep understanding of how well we're solving those problems. This workshop offers an overview of research best practices in order to align the team prior to the first stakeholder or user interviews they conduct.
```

Comment:
Continuous research helps ensure that we’re building something of value at each step. Needs and problems live with people, so in order to create value we need a deep understanding of how well we're solving those problems. This session offers an overview of research best practices in order to align the team prior to the first stakeholder or user interviews they conduct.

Reply:
Updated.